ios/Exponent/Versioned/Core/Api/EXNotifications.m

```diff
@@ -92,41 +93,116 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId kernelServiceDeleg
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(__unused RCTPromiseRejectBlock)reject)
 {
-  UILocalNotification *notification = [self _localNotificationFromPayload:payload];
 
+  UNMutableNotificationContent* content = [self _localNotificationFromPayload:payload];
   [EXUtil performSynchronouslyOnMainThread:^{
-    [RCTSharedApplication() presentLocalNotificationNow:notification];
+    UNTimeIntervalNotificationTrigger* trigger = [UNTimeIntervalNotificationTrigger triggerWithTimeInterval:1 repeats:NO];
+    UNNotificationRequest* request = [UNNotificationRequest
+                                      requestWithIdentifier:content.userInfo[@"id"] content:content trigger:trigger];
+    [[UNUserNotificationCenter currentNotificationCenter] addNotificationRequest:request withCompletionHandler:^(NSError * _Nullable error) {
+      if (error != nil) {
+        NSLog(@"%@", error.localizedDescription);
+        reject(@"Could not make notification request", error.localizedDescription, error);
```

Comment:
First argument to `reject` should be error code, second — message and third — an error, right?

I think `E_NOTIF` would work better as the code and
`Could not add a notification request: ${error.description}` as the message.